roles/edpm_libvirt/templates/virtproxyd.conf

```diff
@@ -2,5 +2,10 @@ unix_sock_ro_perms="0444"
 unix_sock_rw_perms="0770"
 auth_unix_ro="none"
 auth_unix_rw="none"
+auth_tls={% if edpm_libvirt_sasl_auth_enabled|bool %}
```

Comment:
as noted above i would prefer to use edpm_libvirt_tls_certs_enabled here
since we don't want to allow using sasl auth indepently of tls.

Reply:
doesn't need to be conditional, auth_tls will be ignored if TLS isn't enabled